### ISSUE TITLE
Option to load a source in the preview player

### ISSUE THREAD

**Issue by forbesjo** [videojs/videojs.com#84]
Now that HLS/DASH support is built into video.js it might be worth allowing users to load a custom source into the demo player

**Comment by gkatsev** [comment]
Yeah, the quick answer is to add a box to the advanced page. Though, we may want to redesign the main page a bit as well.

**Comment by gkatsev** [comment]
Also, we can transcode oceans into hls and dash and use that as defaults as well.
